core-plugins/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -291,3 +291,12 @@ quotedValueDelimitedTestFiles=dummy
 fileSinkTargetBucket=file-plugin-output
 recursiveTest=dummy
 testOnCdap=true
+
+## EXCEL-PLUGIN-PROPERTIES-START ##
+excelTestFile=dummy
+excelFile=testdata/exceldata/testExcel.xlsx
+Pattern=.*\.xlsx$
```

Comment:
Does the plugin only support `xlsx` format? 

Or will the tests for other formats come later?

Reply:
yes this only support xls format. And as per ITN's we have to add for this one only.

Reply:
`xls` and `xlsx` are different formats? Which one are we looking to add the test for?